components/Navigation.js

```diff
@@ -47,30 +49,39 @@ export default function Navigation({
       {!isNavigationOpen && (
         <>
           <StyledSearchMenu onClick={toggleSearchBar}>
-          <Image src={"/icons/search.png"} width={35}  height={35} alt= {"search icon"}/>
+            <Image
+              src={"/icons/search.png"}
```

Comment:
Consider to remove the curly braces around the string literals as they are unnecessary when you are directly using a string. This cleans up the code slightly. 

```suggestion
              src="/icons/search.png"
```